static-src/scss/core/_hero-unit.scss

```diff
@@ -49,6 +49,38 @@
     .button-group {
         @include rem(margin-top, 20px);
     }
+
+    &.phase-PhaseRefinement {
```

Comment:
ah, got it.  sorry.

(it was, but it needs to be in a more general place now.)